docker/wiremock/stubs/__files/correctSimpleTextDocAttachment.json

```diff
@@ -0,0 +1,6 @@
+{
+  "resourceType": "Binary",
+  "id": "care-record-document--30e49be4-42aa-4dff-87d0-3c1c2cc22a6f",
+  "contentType": "text/plain",
+  "content": "SG93IGFyIHlvdQ=="
```

Comment:
This content type looks to be Base64.

I don't think in the scenario the Document was encoded in Base64.

Might be worth having a quick call to understand what/how you tested this scenario?!

Reply:
As per the specification retrieved attachment documents are expected to be encoded with binary Base64. 
https://developer.nhs.uk/apis/gpconnect-1-6-0/access_documents_development_binary.html